tests/phpunit/tests/strings-in-block/test-html.php

```diff
@@ -58,12 +58,13 @@ public function it_finds_paragraph() {
 
 	/**
 	 * @test
+	 * @group wpmlcore-6661
 	 */
 	public function it_does_not_find_column_if_there_is_no_xpath() {
 
 		$config_option = \Mockery::mock( 'WPML_Gutenberg_Config_Option' );
 		$config_option->shouldReceive( 'get' )
-		              ->andReturn( array( 'core/column' => array( 'xpath' => array() ) ) );
+		              ->andReturn( array( 'core/column' => array() ) );
```

Comment:
@strategio I would extend the test method name.

Right now, it stays "it does not find column if there is no xpath". This assumes that:
- the reader knows what a "column" is
- that we are dealing with the "column" block
- that "xpath" is in the block's configuration

I suggest you use either "it does not find the content if the block configuration has no xpath" or at least (though still incomplete) "it does not find the column block if there is no xpath".

Reply:
Renamed to `it_does_not_find_the_column_content_if_the_block_configuration_has_no_xpath`.